todo_app/classes.py

```diff
@@ -0,0 +1,20 @@
+from todo_app.enums import List_Ids, Status
+
+def map_column_id_to_status(column_id: str):
+    match column_id:
+        case List_Ids.TO_DO_LIST.value:
```

Comment:
Given you use the value here, are we actually gaining much over just referencing the environment variable directly?
```python
        case os.getenv("TO_DO_LIST_ID"):
```